SimG4CMS/Forward/src/TotemSD.cc

```diff
@@ -49,7 +50,7 @@ TotemSD::TotemSD(const std::string& name,
                  const SensitiveDetectorCatalog& clg,
                  edm::ParameterSet const& p,
                  const SimTrackManager* manager)
-    : SensitiveTkDetector(name, es, clg, p),
+    : SensitiveTkDetector(name, clg),
```

Comment:
Why some SD classes, you keep parameterset p (e.g. TotemT2ScintSD.cc), while some not (e.g. this one)?

Reply:
Please ignore, I think I can follow it now.

Reply:
As you've already figured out, I kept only those constructor arguments which are actually used by the constructor of the class.